src/components/v5/frame/NavigationSidebar/partials/NavigationSidebarSecondLevel/NavigationSidebarSecondLevel.tsx

```diff
@@ -3,7 +3,7 @@ import clsx from 'clsx';
 import React, { type FC } from 'react';
 
 import { useTablet } from '~hooks/index.ts';
-import { multiLineTextEllipsis } from '~utils/strings/index.ts';
+import { multiLineTextEllipsis } from '~utils/strings.ts';
```

Comment:
Did you actually refactored all these barrel files in this PR or this some rebase nonesense ?

Reply:
It was intentional, I saw that the `strings` folder only had 1 file which was the `index` so I just removed the folder and renamed the `index` to `strings`